dashboard.py

```diff
@@ -648,8 +648,12 @@ def main() -> None:
     for runtime in runtimes:
         make_timeseries_html_report(df, output_dir, runtime)
 
-    # Do not use data that is more than a week old in statistical analysis
-    df_recent = df[df["end"] > df["end"].max() - pandas.Timedelta("7d")]
+    # Do not use data that is more than a week old in statistical analysis.
+    # Also exclude failed tests.
+    df_recent = df[
+        (df["end"] > df["end"].max() - pandas.Timedelta("7d"))
+        & (df["call_outcome"] == "passed")
+    ]
```

Comment:
The purpose of these A/B tests is to compare performance, not failure rates. For those there's already the red light on the CI build.

Reply:
Hm, but sometimes changes in performance can cause failures. In those cases, this could give a distorted view of a change as being purely good for performance, when it actually might just increase the _variability_ of performance to the point where some of the bad runs are failures.

At the same time, I agree that we usually don't want results from failed tests mixed in. I guess this makes more sense to merge right now, but it worries me a bit to not have an easy option to see both with and without failures.